test/watcher.js

```diff
@@ -20,7 +20,7 @@ test('chokidar is not installed', function (t) {
 	});
 
 	try {
-		subject.start({}, {files: [], excludePatterns: []}, []);
+		subject.start({}, {}, [], []);
```

Comment:
Would be nice to keep `{exludePatterns: []}` here, even though the test doesn't fail without it.